src/main/java/com/codeborne/selenide/appium/SelenideAppiumTargetLocator.java

```diff
@@ -0,0 +1,26 @@
+package com.codeborne.selenide.appium;
+
+import com.codeborne.selenide.WebDriverRunner;
+import com.codeborne.selenide.logevents.SelenideLogger;
+import org.openqa.selenium.ContextAware;
+
+import java.util.Set;
+
+public class SelenideAppiumTargetLocator {
```

Comment:
Why do we need this separate class? Why cannot this code be just in `SelenideAppium.java`?

Reply:
Thought of moving implementations to a separate class. Should I rewrite in the SelenideAppium class?